detox/test/integration/e2e/passing-sandbox.js

```diff
@@ -0,0 +1,9 @@
+const { config, session } = require('detox/internals');
+const { default: expect } = require('expect');
+
+describe('Sandbox', () => {
+  test('detox/internals should be accessible from the sandbox', () => {
+    expect(session.testSessionIndex).toBe(0);
```

Comment:
This is a rather basic expectation set for the Detox Internals API being accessible from the Jest sandbox.

On the other hand, I don't oblige myself to make it thorough as the goal of this pull request was only to enable that sandbox access it self. And this test exactly does the job.